guides/release/tutorial/14.md

```diff
@@ -0,0 +1,83 @@
+
+Here, we are going to do just that! We are going to implementing the "View Larger" and "View Smaller" functionality. Specifically, we want to give our users the ability to view a larger version of the property's image by clicking on it, and when they are done, they can click on it again to go back to the smaller version.
+
+In other words, we want a way to _toggle_ the image between one of the two *[states](TODO: link to states)*. In order to do that, we need a way for the component to store two potential states, and to also be aware which of the two possible states it is currently in.
+
+Ember allows us to optionally associate JavaScript code with a component for exactly this purpose. We can generate a JavaScript file for our `<Rental::Image>` component by running the `component-class` generator:
+
+```
+ember generate component-class rental/image
+```
+
+This generated a JavaScript file with the same name as our component's template at `app/components/rentals/image.js`. It contains a *[JavaScript class](TODO: link to JavaScript class)*, subclassing from `@glimmer/component`.
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        <p>`@glimmer/component`, or *[Glimmer component](TODO: link to Glimmer component)*, is one of the several component classes available to use. Component classes are a great starting point whenever you want to add behavior to your components. In this tutorial, we will be using Glimmer components exclusively.</p>
+        <p>In general, Glimmer components should be used whenever possible. However, you may also see `@ember/components`, or *[classic components](TODO: link to classic components)*, being used in older apps. You can tell them apart from looking at their import path, which is helpful for looking up the respective documentation, as they have different and incompatible APIs.</p>
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="Ember Mascot">
+  </div>
+</div>
+
+Ember will create an *[instance](TODO: link to instance)* of the class whenever our component is invoked. We can use that to store our state:
+
+```js {data-filename="app/components/rental/image.js" data-diff="+4,+5,+6"}
+import Component from '@glimmer/component';
+
+export default class RentalImageComponent extends Component {
+  constructor(...args) {
+    super(...args);
+    this.isLarge = false;
+  }
+}
+```
+
+Here, in the *[component's constructor](todo://link to component's constructor)*, we *[initialized](TODO: link to initialized)* the *[instance variable](TODO: link to instance variable)* `this.isLarge` to the value `false`, since this is the default state that we want for our component.
```

Comment:
```suggestion
Here, in the *[component's constructor](todo://link to component's constructor)*, we *[initialized](TODO: link to initialized)* the *[instance variable](TODO: link to instance variable)* `this.isLarge` with the value `false`, since this is the default state that we want for our component.
```